certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk7809/CodeSystemFunktionsnedsattning.java

```diff
@@ -0,0 +1,65 @@
+package se.inera.intyg.certificateservice.infrastructure.certificatemodel.fk7809;
+
+import se.inera.intyg.certificateservice.domain.common.model.Code;
+
+public class CodeSystemFunktionsnedsattning {
+
+  public static final String CODE_SYSTEM = "FUNKTIONSNEDSATTNING";
```

Comment:
Since these codes does not have a codesystem realated to them, does it make sense to set it to null instead?

Reply:
I think having it symbolizes it's a code system we set up and not FK